paymentsheet/src/main/java/com/stripe/android/customersheet/StripeCustomerAdapter.kt

```diff
@@ -50,13 +49,9 @@ internal class StripeCustomerAdapter @Inject constructor(
                     id = customerEphemeralKey.customerId,
                     ephemeralKeySecret = customerEphemeralKey.ephemeralKey,
                 ),
-                types = listOfNotNull(
```

Comment:
Does this still work with a non US merchant? 

Should we also fetch sepa debit methods? Why would we filter anything different in CustomerSheet than we do in PaymentSheet?

Reply:
We only support cards and us bank accounts in CustomerSheet. Yes, this works with non-us merchants.